cmd/machine-config-operator/bootstrap.go

```diff
@@ -79,6 +79,8 @@ func init() {
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.haproxyImage, "haproxy-image", "", "Image for haproxy.")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.baremetalRuntimeCfgImage, "baremetal-runtimecfg-image", "", "Image for baremetal-runtimecfg.")
 	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.oauthProxyImage, "oauth-proxy-image", "", "Image for origin oauth proxy.")
+	bootstrapCmd.PersistentFlags().StringVar(&bootstrapOpts.baseOSContainerImage, "baseos-image", "", "Image for rhel-coreos-8")
```

Comment:
To make sure, in the hypershift case, `bootstrapOpts.imageReferences` would be empty, bypassing the whole write below, and so we can just specify it here instead?

I am +1 to this if we don't get the imagestream stuff sorted out

Reply:
Right, that's what I think should happen but I didn't test it.